Makefile

```diff
@@ -12,7 +12,10 @@ bazel-generate:
 	SYNC_VENDOR=true hack/dockerized "./hack/bazel-generate.sh"
 
 bazel-build:
-	hack/dockerized "hack/bazel-fmt.sh && hack/bazel-build.sh"
+	hack/dockerized "bazel build \
+		--platforms=@io_bazel_rules_go//go/toolchain:linux_ppc64le \
```

Comment:
I think that we should start moving `--platforms` and `--workspace_status_command` into a .bazelrc and select them based on a config. @cynepco3hahue what do you think?

Reply:
@rmohr It can be cool, but I thought that `.bazelrc` used mostly to set the default for bazel commands.
Do you have an example of how can we select them by config?

Reply:
here is an example: https://docs.bazel.build/versions/master/guide.html#bazelrc.

Reply:
thanks for the link, but we anyway will need to provide `--config` parameter for the bazel CLI command, so I think we can move `--workspace_status_command` under the `.bazelrc` but leave the platform for the visibility or we anyaway use shell scripts that includes bazel command so we can on runtime to decide which platform do we use and apply the right platform to the bazel CLI command.